packages/store/src/decorators/state.ts

```diff
@@ -1,16 +1,21 @@
-import { ensureStoreMetadata, MetaDataModel, StateClass } from '../internal/internals';
+import {
+  ensureStoreMetadata,
+  MetaDataModel,
+  MutateMetaOptions,
+  StateClass
+} from '../internal/internals';
 import { META_KEY, META_OPTIONS_KEY, StoreOptions } from '../symbols';
 import { StoreValidators } from '../utils/store-validators';
 
 /**
  * Decorates a class with ngxs state information.
  */
 export function State<T>(options: StoreOptions<T>) {
-  function mutateMetaData(
-    meta: MetaDataModel,
-    stateClass: StateClass,
-    optionsWithInheritance: StoreOptions<T>
-  ) {
+  function mutateMetaData({
+    meta,
+    stateClass,
+    optionsWithInheritance
+  }: MutateMetaOptions<T>): void {
```

Comment:
Because we don't use void on a regular basis, I think we should avoid adding that.

Reply:
I think it's time for us to use return types everywhere in the future to configure tslint